### PR TITLE
PP 10667 GHP preview production pipeline

### PR DESCRIPTION
**Pull request by nlsteers**
### WHAT 

- GHA to trigger visual regression testing if the source directory has been modified
  - this workflow is configured to only run when the source directory has been modified. Otherwise, we're generating snapshots for content identical to the base build
- percy configuration 
  - exclusions are in place for pages we strictly redirect from

### CODE REVIEW

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -43,10 +45,21 @@ jobs:
           bundle exec middleman build
       - name: Setup Pages
         uses: actions/configure-pages@37538b280cbfab35da9dcd936a2b95f1f522d927
+      - name: Prepare archive for upload
+        shell: sh
+        run: |
+          chmod -c -R +rX "build"
```
Comment (oswaldquek) [inline .github/workflows/static.yml:51]: Hm what does this line do? There's no "build" directory, and running it locally I get 
```
> chmod -c -R +rX "build"
chmod: illegal option -- c
```

**Review by oswaldquek — COMMENTED** [review]
Just one comment. 
Is it possible to see an output of a percy run, or is that not possible until this PR is merged?

File: .github/workflows/static.yml (hunk 1 above)
Comment (nlsteers) [reply .github/workflows/static.yml:51]: This fixes any directory permissions prior to archiving, "build" is generated by middleman earlier in the workflow.

`chmod` might behave differently on your Mac versus the action runner which is Ubuntu.

**Comment by nlsteers** [comment]
> Just one comment. Is it possible to see an output of a percy run, or is that not possible until this PR is merged?

I've invited you to the team on Percy, you can then see the visual diff by clicking the `details` link on the percy check

**Review by oswaldquek — APPROVED** [review]

**Review by oswaldquek — APPROVED** [review]

**Pull request merged**